packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -246,6 +249,23 @@ class _BottomSheetState extends State<BottomSheet> {
     return false;
   }
 
+  @override
+  void dispose() {
+    _effectiveAnimationController.dispose();
```

Comment:
You should only dispose this if you own this controller. If it was passed in to you, it should not be disposed.